hip-0000-registry-contract-for-naming-services.md

```diff
@@ -0,0 +1,160 @@
+---
+hip: <HIP number (this is determined by the HIP editor)>
+title: REGISTRY CONTRACT containing default naming structure format for HTLDs & SLDs ever minted over Hedera network
+author: som-web23,(somkirann@web23.io)
+working-group: (somkirann@web23.io),(rahul@web23.io),(del@web23.io)
+type: Process
+category: Service
+needs-council-approval: No
+status: Draft
+created: 2022-07-29 
+updated: 2022-07-29
+---
+
+## Table of Contents
+
+1.	Abstract
+2.	Motivation
+3.	Rationale
+4.	Specification
+5.	Detailed note on the flow
+6.	Reference Implementation
+7.	Rejected Ideas
+8.	Open Issues
+9.	References
+10.	Copyright/license 
+
+
+## Abstract
+
+Fostering overall growth of Hedera, minting of HTLDs (Hedera based -Top-Level-Domains) & SLDs (Second-Level-Domains) within Hedera are open. Any entity, person could mint of the HTLDs & SLDs which could eventually create issues like:
+a.	Duplication of TLDs; .hbar (for example) by one contract & .hbarby the other contract
+b.	Duplication of SLDs; for example, minted SLD user.tld by one contract & user.tld by the other contract
+c.	Phishing attempts and inconvenience amongst the community owing to the duplication of names
```

Comment:
Inconvenience amongst the community is a valid concern here. Creating standardization is necessary, but it should be clear that the community collaboration in standards on this topic will not thwart phishing attempts, the language of "phishing" should be removed as this HIP will appear compromised when phishing continues to happen in adjacent ways.